src/etl/ReportingETL.cpp

```diff
@@ -1008,7 +1008,7 @@ ReportingETL::loadCache(uint32_t seq)
                         backend_->cache().update(res.objects, seq, true);
                         if (!res.cursor || (end && *(res.cursor) > *end))
                             break;
-                        BOOST_LOG_TRIVIAL(debug)
+                        BOOST_LOG_TRIVIAL(trace)
```

Comment:
I think we still want some sort of progress marker around the cache download at the debug level. It doesn't have to be as often as this was, but the download can take 45 minutes, and its nice to have a way to see in the logs how far along it is.

Reply:
Thinking about it more, how does the output look if we change this to trace, but the statement outside the loop is left at debug?

Reply:
It looks like every 3-4 seconds or so we get `Finished a cursor` followed by `Starting a cursor` (both are long lines). It's quite a lot of logs. The `Finished a cursor` is on `info` level and `Starting a cursor` is on `debug` as you suggested. I'd leave  this on trace and probably push `Finished a cursor` down to `debug` too but it`s up to you. If we leave just the `Finished` on `info` maybe it's a good progress marker for the download. wdyt?